src/api/usersRouter.ts

```diff
@@ -29,42 +34,38 @@ usersRouter.post('/', async (req, res) => {
   res.status(201).json(user);
 });
 
-const userIdParser = z.object({
-  userId: z.string(),
+usersRouter.put('/current/acceptTerms', enrichCurrentUser, async (req: UserRequest, res) => {
```

Comment:
I think our convention was to to use the same router as /:userId and parse the param to be string `current`?

Reply:
That's what we did for participant, but that wouldn't make sense here - I don't want this route to work for anything except the current user. A user can't accept the terms and conditions for a different user.